ivy/functional/ivy/meta.py

```diff
@@ -530,25 +530,29 @@ def fomaml_step(
 
 
 @handle_exceptions
+@handle_array_function
+from typing import Callable, List
+import ivy
+import ivy.numpy as np
+
+
 @handle_array_function
 def reptile_step(
     batch: ivy.Container,
-    cost_fn: Callable[[ivy.Container, ivy.Container], ivy.Array],
+    cost_fn: Callable,
     variables: ivy.Container,
     inner_grad_steps: int,
     inner_learning_rate: float,
     /,
     *,
-    inner_optimization_step: Callable[
-        [Callable[[ivy.Array, ivy.Container], ivy.Array], ivy.Container, float],
-        ivy.Container,
-    ] = gradient_descent_update,
+    inner_optimization_step: Callable = gradient_descent_update,
     batched: bool = True,
     return_inner_v: Union[str, bool] = False,
     num_tasks: Optional[int] = None,
     stop_gradients: bool = True,
-) -> Tuple[ivy.Array, ivy.Container, Any]:
+):
     """
+<<<<<<< HEAD
```

Comment:
I think something has gone wrong with your merge conflict resolution - this HEAD should have been removed